lampo-vls/src/protocol_handler.rs

```diff
@@ -0,0 +1,77 @@
+use lampo_common::{
+    bitcoin::Address,
+    vls::{
+        proxy::{
+            vls_protocol_client::{Error, Transport},
+            vls_protocol_signer::{
+                handler::{Handler, HandlerBuilder, RootHandler},
+                vls_protocol::{model::PubKey, msgs},
+            },
+        },
+        signer::{
+            bitcoin::Network,
+            node::NodeServices,
+            persist::DummyPersister,
+            policy::simple_validator::{make_simple_policy, SimpleValidatorFactory},
+            signer::ClockStartingTimeFactory,
+            util::{clock::StandardClock, crypto_utils::generate_seed},
+        },
+    },
+};
```

Comment:
This is quite horrible :) probably you fix it in a future commit but this is horrible to have in the git history